notebooks/usage.ipynb

```diff
@@ -43,36 +43,70 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "\n",
+    "sys.path.append(str(Path.home() / Path(\"Development\") / \"iSparrow\"))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys \n",
-    "sys.path.append(\"..\")\n",
-    "import iSparrow.sparrow_model_base as spm\n",
-    "import iSparrow.sparrow_recording as spr\n",
-    "import iSparrow.preprocessor_base as spb\n",
+    "from iSparrow import PreprocessorBase\n",
```

Comment:
There's an explicit remark above the code lines that the package must be installed first from the repo. Then it should work. (later it will be installed via pip anyway, right? )